tests/gtest/avifallocationtest.cc

```diff
@@ -188,5 +188,17 @@ TEST(EncodingTest, MaximumInvalidDimensions) {
                AVIF_RESULT_UNSUPPORTED_DEPTH);
 }
 
+TEST(AvifAllocTest, Extremes) {
+  void* p0 = avifAlloc(0);
```

Comment:
https://man7.org/linux/man-pages/man3/malloc.3.html

*Nonportable behavior
       The behavior of these functions when the requested size is zero
       is glibc specific; other implementations may return NULL without
       setting errno, and portable POSIX programs should tolerate such
       behavior.  See realloc(3p).*